src/changelog/sources/dependabot/source_test.go

```diff
@@ -40,134 +40,140 @@ func TestSource_Source(t *testing.T) {
 		expected []changelog.Dependency
 	}{
 		{
-			commit: git.Commit{Message: "Whatever actions/github-script from 1.0.2 to 1.0.4."},
+			commit: git.Commit{Message: "Whatever actions/github-script from 1.0.2 to 1.0.4.", Author: "dependabot[bot]"},
```

Comment:
To make the test readable, we can "cheat" a bit and, in the test body, set `Author` to a default value if it's empty.

Reply:
done